webapp/src/FosRestBundle/FlattenExceptionHandler.php

```diff
@@ -7,19 +7,20 @@
 use JMS\Serializer\GraphNavigatorInterface;
 use JMS\Serializer\Handler\SubscribingHandlerInterface;
 use JMS\Serializer\JsonSerializationVisitor;
+use Symfony\Component\DependencyInjection\Attribute\Autoconfigure;
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
 use Symfony\Component\ErrorHandler\Exception\FlattenException;
 use Symfony\Component\HttpFoundation\Response;
 
-/**
- * Class FlattenExceptionHandler
- *
- * @package App\Controller\API
- */
 class FlattenExceptionHandler implements SubscribingHandlerInterface
 {
-    public function __construct(private readonly ExceptionValueMap $messagesMap, private readonly bool $debug)
-    {
-    }
+    public function __construct(
+        #[Autowire(service: 'fos_rest.exception.messages_map')]
```

Comment:
Why do we have to pass these Autowire annotations now? It seems that previously that was done by *magic*

Reply:
The magic was in services.yaml and you needed to know EXACTLY how to call your variable. I prefer to put the logic close to the code but if others disagree we can do the old thing again

Reply:
SGTM